backend/dataall/core/metadata_manager/metadata_form_entity_manager.py

```diff
@@ -0,0 +1,60 @@
+from abc import ABC
+from typing import Dict
+
+from dataall.base.api import GraphQLEnumMapper
+
+
+class MetadataFormEntityTypes(GraphQLEnumMapper):
+    Organizations = 'Organization'
+    OrganizationTeams = 'Organization Team'
+    Environments = 'Environment'
+    EnvironmentTeams = 'Environment Team'
+    S3Datasets = 'S3-Dataset'
+    RDDatasets = 'Redshift-Dataset'
+    Worksheets = 'Worksheets'
+    Dashboards = 'Dashboard'
+    ConsumptionRoles = 'Consumption Role'
+    Notebooks = 'Notebook'
+    MLStudioEntities = 'ML Studio Entity'
```

Comment:
nit: rename `MLStudioEntities` to `MLStudioUser`
nit2: some names are in plural, other in singular